embedded-service-registry/readme.md

```diff
@@ -0,0 +1,58 @@
+# Service registry in Azure Spring Apps
+
+This sample shows how to use embedded service registry in `Azure Spring Apps`.
```

Comment:
how about `cloud native service registry`? embedded is very similar to manged, which means eureka in our case